midterm-2023-11-07/code/5263-angela-kostadinova/script.ts

```diff
@@ -0,0 +1,106 @@
+interface Movie {
```

Comment:
The interface is more in line with what will be shown on the screen, not with what we're getting from the data. +5